BSB_LAN/BSB_LAN.ino

```diff
@@ -5200,7 +5240,7 @@ void loop() {
           uint8_t found_ids[10] = { 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF };
           if (bus->Send(TYPE_QINF, 0x053D0002, msg, tx_msg, NULL, 0, false)) {
             unsigned long startquery = millis();
-            while (millis() - startquery < 10000) {
+            while (millis() - startquery < 10000 && millis() >= startquery) {
```

Comment:
millis(). It is unlikely that the /Q command will be called on day 49, but still.